qcsrc/localized_text.qc

```diff
@@ -33,7 +33,7 @@
 
 #ifdef __LIBREQUAKE__
 
-#define LOC_EPILOGUE_0              "\nAs the ethereal essence of your very\nbeing drifts through time and space,\nyou experience relentless calm and\nserene determination. You embrace the\nliminal, formless nature of your being\nwithin the amorphous expanse between\nrealities.\n\nSuspended between the veils of life\nand death, among the realm of celestial\ndissolution, you wait to be called upon\nagain. To regain your corporeal form\nand step into the crucible of fate.\nTo finish your task and keep at bay the\nonslaught of cosmic horrors.\n"
+#define LOC_EPILOGUE_0              "\nAs the ethereal essence of your very\nbeing drifts through time and space,\nyou experience relentless calm and\nserene determination. You embrace the\nliminal, formless nature of your existence\nwithin the amorphous expanse between\nrealities.\n\nSuspended between the veils of life\nand death, among the realm of celestial\ndissolution, you wait to be called upon\nagain. To regain your corporeal form\nand step into the crucible of fate.\nTo finish your task and keep at bay the\nonslaught of cosmic horrors.\n"
```

Comment:
Looks like the engine truncates the line with "existence". Move "existence" into the next line and "between" into the last line of the paragraph?